data/meta/tokens/1/0x051d7e5609917Bd9b73f04BAc0DED8Dd46a74301.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sBTCv2 Pool",
+  "symbol": "crvWSBTC",
+  "description": "This token represents a Curve liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This liquidity pool contains sBTC and wBTC.",
```

Comment:
`This liquidity pool contains sBTC and wBTC and serves as a base pool for other Bitcoin pools.`

Reply:
fixed